lib/fluent/plugin/in_tail.rb

```diff
@@ -211,7 +213,14 @@ def start
         FileUtils.mkdir_p(pos_file_dir) unless Dir.exist?(pos_file_dir)
         @pf_file = File.open(@pos_file, File::RDWR|File::CREAT|File::BINARY, @file_perm)
         @pf_file.sync = true
-        @pf = PositionFile.parse(@pf_file)
+        @pf = PositionFile.load(@pf_file, logger: $log)
```

Comment:
Can we use `log` method as logger here?
I want to switch log level within PositionFile class.

Reply:
fixed https://github.com/fluent/fluentd/pull/2805/commits/0ffdd845d39d6f6d1fe28638edc474fa7177768b